recipe/meta.yaml

```diff
@@ -9,7 +9,7 @@ source:
   sha256: e0731a60ba540cd19bbbefe771a9076dcd2dde90713a8f87f27f53f2d1db7727
 
 build:
-  number: 0
+  number: 1
   script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
```

Comment:
Can we make this one noarch?

Reply:
I don't think so, due to the post-link scripts.